addons/l10n_es_edi_sii/models/account_move.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    l10n_es_edi_is_required = fields.Boolean(
+        string="Is the Spanish EDI needed",
+        compute='_compute_l10n_es_edi_is_required'
+    )
+    l10n_es_edi_csv = fields.Char(string="CSV return code", copy=False)
+    l10n_es_registration_date = fields.Date(
+        string="Registration Date",
+        help="Technical field to keep the date the invoice was sent the first time as the date the invoice was "
+             "registered into the system.",
+    )
+
+    # -------------------------------------------------------------------------
+    # COMPUTE METHODS
+    # -------------------------------------------------------------------------
+
+    @api.depends('move_type', 'company_id')
+    def _compute_l10n_es_edi_is_required(self):
+        for move in self:
+            move.l10n_es_edi_is_required = move.is_invoice() and move.country_code == 'ES'
```

Comment:
This is not correct, as you can have several companies, but only few of them requiring SII.